packages/devextreme/js/__internal/ui/html_editor/m_html_editor.ts

```diff
@@ -235,29 +230,14 @@ const HtmlEditor = Editor.inherit({
   },
 
   _updateContainerMarkup() {
-    let markup = this.option('value');
-
-    if (this._isMarkdownValue()) {
-      this._prepareMarkdownConverter();
-      markup = this._markdownConverter.toHtml(markup);
-    }
+    const { value } = this.option();
 
-    if (markup) {
-      const sanitizedMarkup = this._removeXSSVulnerableHtml(markup);
+    if (value) {
+      const sanitizedMarkup = this._removeXSSVulnerableHtml(value);
       this._$htmlContainer.html(sanitizedMarkup);
     }
   },
 
-  _prepareMarkdownConverter() {
-    const MarkdownConverter = ConverterController.getConverter('markdown');
-
-    if (MarkdownConverter) {
-      this._markdownConverter = new MarkdownConverter();
-    } else {
-      throw Errors.Error('E1051', 'markdown');
```

Comment:
Should we leave the error "E1051"? https://github.com/EugeniyKiyashko/DevExtreme/blob/454d2d9912480f3d0d21aa7433d6b476998bf8d8/packages/devextreme/js/ui/widget/ui.errors.js#L202

Reply:
[6ea44e4acfef4eacad69cbe86a07333f26a7b461]